lando/server/tests/test_cwlworkflow.py

```diff
@@ -25,6 +26,10 @@
     type: stdout
 """
 
+# Temp directory in users home directory needed to run cwl on OSX
+USERNAME = pwd.getpwuid(os.getuid())[0]
+USER_TEMP_DIR = "/Users/" + USERNAME + "/Documents/"
```

Comment:
Why not look up home directory with `os.getenv('HOME')` instead of getting uid, username, and then prefixing with the platform-specific `/Users`?